kie-api/src/main/java/org/kie/api/KieServices.java

```diff
@@ -316,9 +316,16 @@
     Environment newEnvironment();
 
     /**
+     * Returns a reference to the KieServices singleton
+     */
+    static KieServices get() {
+        return Factory.get();
+    }
+
+    /**
      * A Factory for this KieServices
      */
-    public static class Factory {
+    class Factory {
```

Comment:
While I think this is not related to this PR and I welcome the added method above, I would like to have double-confirmation this is a (/another) breaking change in the Kie API, correct?